source/app.css

```diff
@@ -0,0 +1,53 @@
+cloudflare-app[app-id="a-better-browser"] {
```

Comment:
What if we release the app with two versions, an older one we use to grandfather in old installs, and a new one for new installers.  If the grandfathered in installers chose to upgrade they can, but if they would rather not they can remain on the older version indefinitely.